etc/db-config.yaml

```diff
@@ -303,3 +303,8 @@
             public: false
             description: URL of an ICPC iCAT instance if such is available; will be linked to from the submission verification box.
             docdescription: See https://github.com/icpc-live/autoanalyst
+        -   name: external_contest_source_critical
+            type: int
+            default_value: 120
+            public: false
+            description: Time in seconds after an external contest source reader last checked in before showing its status as "critical".
```

Comment:
Aren't we the one checking in?

Reply:
I don't follow? The 'external contest source reader' is checking in.

Reply:
No? I still don't follow. This number is used like this:
If the daemon running the feed importer didn't report itself for more than this value, alert like crazy.